### ISSUE TITLE
'gbk' codec can't decode byte 0x80

### ISSUE THREAD

**Issue by skill4net** [Shail-Shouryya/yt_videos_list#3]
File "C:\Anaconda3\lib\site-packages\yt_videos_list\file\update_file.py", line 11, in store_already_written_videos
    if file_type == 'csv':       return set(re.findall(r'(https://www\.youtube\.com/watch\?v=.+?),', file.read()))
UnicodeDecodeError: 'gbk' codec can't decode byte 0x80 in position 90: illegal multibyte sequence

This lib is good when using python prompt to run for the Chinese channel name, however, when placed into code, the above error display.
Please try to use encoding="utf-8"

for example: with open(file) as f:  ==> with open(file, ‘r’, encoding=‘utf-8’) as f:

**Comment by Shail-Shouryya** [comment]
Taking a look 🙂

Do you have a particular channel you were running it on when it broke? If you do, I can test it on that to make sure it's fixed!

**Comment by Shail-Shouryya** [comment]
Added a fix in the commit above ↑

Can you run the following to see if the commit fixed the problem you were seeing?
```
git clone https://github.com/Shail-Shouryya/yt_videos_list.git
cd yt_videos_list\python
git checkout issue3
pip install .

python
from yt_videos_list import ListCreator

lc = ListCreator()
lc.create_list_for(url='the_channel_url')
```

**Comment by Shail-Shouryya** [comment]
Merged the corresponding PR (#5) for this issue since I didn't see any activity on this issue nor on [#5](https://github.com/Shail-Shouryya/yt_videos_list/pull/5).

Please reopen this issue if the problem still exists!
